Lib/fontTools/ttLib/tables/C_F_F_.py

```diff
@@ -3,7 +3,6 @@
 from fontTools import cffLib
 from . import DefaultTable
 
-
```

Comment:
why removing the blank line? Top-level classes/functions definitions should be separated by two blank lines in PEP8 style guide.